packages/office-addin-dev-settings/test/unit-tests/test.ts

```diff
@@ -319,28 +319,6 @@ describe("Registration", function() {
         assert.strictEqual(first.id, firstManifestId);
         assert.strictEqual(first.manifestPath, firstRegisteredManifestPath);
       });
-      if (process.platform === "win32") {
```

Comment:
Why did we remove these tests?

Reply:
These test are for the Win32 specific method that uses two arguments allowing for the registry to have a key other than the add-in id in the manifest.  However we currently don't expose anyway to do that directly and the only place we use it we are reading the manifest specifically.  I change the method signature add moved the id reading down a level to be more efficient and so the test no longer makes sense as the method only has a single argument.